modules/fancymeshrenderer/readme.md

```diff
@@ -0,0 +1,3 @@
+# FancyMeshRenderer Module
+
+Provides an extended mesh renderer including different opacity settings, normal estimation methods, texture-based hatching, drawing of silhouettes or edges and illustration-style rendering 
```

Comment:
Would be sweet if we could link to the regression images here so that one would get an understanding of what is possible

Reply:
Probably possible if someone adds reg tests ;)

Reply:
Like the ones already in tests/regression :)

Reply:
They are only for the normals calculation processor, I added them prior to 87b0c2a0e7cd3655b7c8e6913871d6ef976b7728 to ensure my changes didn't break anything. 
But, I was thinking about adding some regression tests, but need to better understand the different options first